### ISSUE TITLE
Issue in scrolling up/down on the page in Dynamics 365

### ISSUE THREAD

**Issue by prachiwalke** [Microsoft/EasyRepro#631]
Can not find the element present at the far bottom or top of the page. It gives NoSuchElementException. Have to manually scroll up to that element while executing the Automation Script. Please help to solve this issue

**Comment by Haywaii** [comment]
Can you provide more details here so people can help? 
Like the code you use, which version of easyRepro, UCI or old interface?

It will be difficult for the community to help without having more information. 

Thanks,

**Comment by ryanpiran** [comment]
@prachiwalke See issue #489 as this seems to be the same issue. A fix was released in the 9.1.0.5230 NuGet package. The issue was caused by the introduction of a 'lazy loading' feature released in a recent update of Dynamics. The fix introduced a new browser option called UCITestMode. When this option is set to True, the UCI app is in test mode. This test mode disables the lazy loading functionality that was introduced. The default behavior will be to set this value to True.

**Comment by mVermaat** [comment]
Yes the TestMode from issue #489 solves the fact that the page loads all elements. However, if the field is underneath the header (red area in image below), I do still get an error (Element Click Intercepted)

Secondary when the value is not in the view and it has to scroll up, it will scroll up making the field be in the red area too, causing an intercept.

![Error](https://user-images.githubusercontent.com/13309338/63340978-7cc72b80-c348-11e9-885c-eee95f38629f.png)

Edit: additional info

**Comment by vcinampudi** [comment]
Seems #631 and  #623 are same
